[Radar] Add explanation of industry and verticals to Radar glossary (#9952)
cloudflare/cloudflare-docs: content/radar/glossary.md

Attacks mitigated by our Level 3 and 4 Denial of Service Attack prevention systems. We show the most used attack vectors as well as the change in attack volume over the selected time range. Selected location is the location of the data center(s) where the attacks were mitigated. Target industry and vertical categories are associated with the customers being attacked.

> Industry categories include businesses connected through their primary activities, such as Information Technology and Services, Retail, or Telecommunications, while vertical categories are high-level groupings that incorporate related industries, such as the "Internet and Telecom" vertical, which includes industries such as "Internet" and "Telecommunications".

Reviewer: ```suggestion
Industry categories include business types grouped by their primary activities, such as information technology and services, retail, or telecommunications. Vertical categories are high-level groupings that incorporate related industries, such as the "Internet and Telecom" vertical, which includes industries such as "Internet" and "Telecommunications".
```

Reviewer: This might benefit from a bit more detail. I'm not sure I understand how we're defining "primary activities" and how that differs from industry verticals.
